### DISCUSSION TITLE
How to Reinitialize on .toNamed()

### DISCUSSION THREAD

**Discussion by sunjianan9900** [jonataslaw/getx#2297] [Q&A]
I have two pages A and B. Page A will initialize a current time and display it in GetxController's onInit(). page B has only one function, which is toNamed(A)
When my open path is.
A - B - A - B - A ...
At this time, the time displayed for any of the later A's is the time of the first one, which means that the later A's are not reinitialized.
This causes an error when repeatedly opening different users' home pages and then going to their posts, always showing the first user's home page.
So, how should I make each A initialize?

**Comment by sunjianan9900** [comment]
does not accept offAll() because I need the user to be able to return to the previously opened pages in order.

**Reply by Shreemanarjun** [reply]
https://github.com/jonataslaw/getx/pull/2155/commits/d6113f3d4afe7ffced82e78ae253249054a3be81 try
